.github/workflows/src/github.js

```diff
@@ -120,3 +120,25 @@ export const CommitStatusState = {
    */
   SUCCESS: "success",
 };
+
+/**
+ * Writes content to the GitHub Actions summary
+ * @param {string} content - Markdown content to add to the summary
+ * @param {typeof import("@actions/core")} core - GitHub Actions core library
+ */
+export async function writeToActionsSummary(content, core) {
+  const summaryFilePath = process.env.GITHUB_STEP_SUMMARY;
```

Comment:
Why are we using an env to pass this information instead of just passing it as a parameter?

Reply:
Looks like that is a github concept. https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions

Can you not use core.summary method instead?

Reply:
Updated the code to use `core.Summary`.